query-engine/js-connectors/js/neon-js-connector/src/neon.ts

```diff
@@ -116,19 +118,19 @@ class PrismaNeon implements Connector, Closeable {
     return this._version
   }
 
-    /**
-   * Run a query against the database, returning the result set.
-   * Should the query fail due to a connection error, the connection is
-   * marked as unhealthy.
-   */
+  /**
+ * Run a query against the database, returning the result set.
+ * Should the query fail due to a connection error, the connection is
+ * marked as unhealthy.
+ */
```

Comment:
Let's ensure the whitespaces remain as they were :)

```suggestion
  /**
   * Run a query against the database, returning the result set.
   * Should the query fail due to a connection error, the connection is
   * marked as unhealthy.
   */
```